x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/AnalyzerTests.java

```diff
@@ -1904,9 +1904,9 @@ public void testLookup() {
             | RENAME languages AS int
             | LOOKUP int_number_names ON int
             """;
-        if (Build.current().isProductionRelease()) {
```

Comment:
We seem to sometimes (rarely) use this check, while we _mostly_ use the `isSnapshot()` one. I _think_ there is a small difference between them and here I tried to have the same approach consistent.